pkgs/tools/typesetting/tex/advi/default.nix

```diff
@@ -34,15 +34,21 @@ ocamlPackages.buildDunePackage rec {
   pname = "advi";
   version = "2.0.0";
 
-  useDune2 = true;
-
   minimalOCamlVersion = "4.11";
 
   src = fetchurl {
     url = "http://advi.inria.fr/advi-${version}.tar.gz";
     hash = "sha256-c0DQHlvdekJyXCxmR4+Ut/njtoCzmqX6hNazNv8PpBQ=";
   };
 
+  postPatch = ''
+    substituteInPlace ./Makefile \
+      --replace "\$(DUNE) install \$(DUNEROOT) --display=short" \
+      "\$(DUNE) install \$(DUNEROOT) --prefix $out --docdir $out/share/doc --mandir $out/share/man"
```

Comment:
I see. I suppose we might as well split `$man` and `$doc` if it is that easy.

Ideally I'd like to try the binary before approving, but I am short on time right now – does advi display correctly?

Reply:
PS: is the fake opam still required?

Reply:
Looks like it displays correctly to me at least
```
nix-shell:~/.cache/nixpkgs-review/pr-226802-2/nixpkgs]$ nix run .#advi -- --help
warning: Git tree '/home/ulrik/.cache/nixpkgs-review/pr-226802-2/nixpkgs' is dirty
usage: /nix/store/1qp5qsx07k20l4xn5lsni2nzqag9avp9-ocaml4.14.1-advi-2.0.0/bin/advi [options] dvifile
  --center-on-cursor   redraw so that cursor is displayed
	
  --debug  General debug
  --debug-pages   debug page motion.
  --debug-positions   Debug positions in source file
  --debug-positions   Debug positions in source file
  --nopostsyncing   do not automatically redraw when syncing if needed 
	
  --scrollfast   scroll to next page instead of page_down 
	
  --showps   ask advi to print to stdout a copy
	of the PostScript program sent to gs.
  --verbose-image-access   change the cursor while loading images,
	(the default is to keep the same cursor).
  --version   print the full Active-DVI version, sub-version and release date.
  -A   ask Active-DVI to use PostScript antialiasing,
	(the default is not to use PostScript antialiasing).
  -ask   set the security policy to "Ask" mode, i.e.
	launching an external application requires explicit confirmation
	(this is the default policy).
  -autoswitch   switch back to master when refreshed
  -bbox   show the bounding box,
	(the default is to hide the bounding box).
  -bgcolor <string>  set the default background color (named, RGB, or X),
	(the default background is "white").
  -browser <com>: set the HTML files viewer command to <com>,
	(the default is "firefox").
  -cache-dir <dir>: set the cache directory to <dir>,
	(the default cache directory is "/tmp").
...
etc
```